### DISCUSSION TITLE
Creating metrics dashboard in SigNoz - page is missing

### DISCUSSION THREAD

**Discussion by shamiulsaurov** [SigNoz/signoz#1387] [Q&A] [answered]
Creating metrics dashboard in SigNoz -- this page is missing. Can you please how to get that?
![image](https://user-images.githubusercontent.com/98252511/178724755-75f9aced-f5e7-4560-8b44-b2483a3e24a3.png)

**Comment by makeavish** [comment, marked as answer]
What's the URL of page?

You can follow this URL to check how to manage dashboards: https://signoz.io/docs/userguide/manage-dashboards/

**Reply by shamiulsaurov** [reply]
URL of the page is https://signoz.io/docs/userguide/dashboards/
